### DISCUSSION TITLE
Does renovatebot run pre-commit/pre-push hooks?

### DISCUSSION THREAD

**Discussion by alvindizon** [renovatebot/renovate#18165] [Request Help] [answered]
In our repo, we have some pre-commit and pre-push hooks that are installed. When making a PR, does renovatebot go through these hooks as well? I couldn't find any indication in the Bitbucket Pipeline logs of this happening. And if it does run hooks, what would happen if an error occurs, i.e., a breaking change happens and lint fails?

**Comment by viceice** [comment, marked as answer]
they are disabled for security reasons. you can enable them when self-hosting renovate

https://docs.renovatebot.com/self-hosted-configuration/#gitnoverify
